lib/CL/clGetPlatformInfo.c

```diff
@@ -218,7 +218,7 @@ POname(clGetPlatformInfo)(cl_platform_id   platform,
       POCL_RETURN_GETINFO (cl_version, pocl_numeric_version);
 
     case CL_PLATFORM_NAME:
-      POCL_RETURN_GETINFO_STR("Portable Computing Language");
+      POCL_RETURN_GETINFO_STR(pocl_get_string_option("POCL_PLATFORM_OVERRIDE", "Portable Computing Language"));
```

Comment:
Please run one of the formatter scripts under tools/scripts/ e.g. clang-format-branch.sh to fix the line length etc.